src/core_plugins/kibana/public/management/app.html

```diff
@@ -14,8 +14,8 @@
 
       <!-- Tabs. -->
       <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
-        <header ng-if="!sectionName" id="tabHeader" tabindex="0">
-          <h2 class="kuiLocalTab">
+        <header ng-if="!sectionName" id="tabHeader">
+          <h2>
```

Comment:
Ah, I know what's wrong. We need to add `margin: 0` to the kuiLocalTab class in the UI Framework. The `<h2>` elements introduces a margin, and this component's design doesn't anticipate this kind of use case.